### ISSUE TITLE
Cannot read property '1' of null

### ISSUE THREAD

**Issue by beorn** [fabiospampinato/enex-dump#1]
Hi, I'm trying to run enex-dump on my 164MB (800+) Evernote notes, but running into a problem:
```console
% wc My\ Notes.enex 
 2109308 2457942 172331592 My Notes.enex

% head -3 My\ Notes.enex
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE en-export SYSTEM "http://xml.evernote.com/pub/evernote-export3.dtd">
<en-export export-date="20181228T065458Z" application="Evernote" version="Evernote Mac 7.7 (457390)">

%  enex-dump --src My\ Notes.enex 

Cannot read property '1' of null
[1]    47307 exit 2     enex-dump --src My\ Notes.enex
```

**Comment by fabiospampinato** [comment]
@beorn could you attach, or send me privately at spampinabio@gmail.com the enex file in question?

**Comment by beorn** [comment]
That's all my notes 🙁  Is there any way to perhaps run with trace/debug mode first to help troubleshoot?

**Comment by fabiospampinato** [comment]
@beorn A few options: 

1. You could try debugging enex-dump yourself, my guess would be that one or more notes in your enex file have a slightly different structure, i.e. some props are missing probably.

2. There are some fixes in the works already [#50](https://github.com/fabiospampinato/notable/issues/50), maybe those will fix your problem too.

3. You could binary-search for the note/notes that causes problems and only send that one to me. Basically try exporting half your notes and check if the problem persists, if it does keep halving the export notes, if it doesn't keep halving the other half.

**Comment by fabiospampinato** [comment]
@beorn I think there's a good chance that this issue has been resolved in v1.3.5 via this commit https://github.com/fabiospampinato/enex-dump/commit/3d99c7f85cc5c05837814869cb4a7d396a92e266, I don't think you could have seen that error any other way 🤔

This means the regex we are using for extracting the alignment isn't good enough for at least one of your notes, it would be useful if you could still find that note and send it to me so that I improve the regex, right now we are kind of ignoring the alignment if the regex doesn't match.

Let me know if the problem has not actually been resolved.

**Comment by beorn** [comment]
Confirming that the problem has been resolved for me!  It ran without a glitch.  Thanks. 🙂
